src/org/usfirst/frc/team467/robot/Grabber.java

```diff
@@ -0,0 +1,45 @@
+package org.usfirst.frc.team467.robot;
+import edu.wpi.first.wpilibj.Spark;
+
+public class Grabber {
+    private static Grabber instance;
+    private Spark left;
+	private Spark right;
+	OpticalSensor os;
+	
+	public final static double GRAB_SPEED = 1.0;
+	public final static double RELEASE_SPEED = -1.0;
+	public final static int GRABBER_L_PORT = 1; 
+    public final static int GRABBER_R_PORT = 2;
```

Comment:
Please rename these from PORT to CHANNEL to match the naming in the Spark class.